index.html

```diff
@@ -1229,28 +1239,58 @@ <h2>Dereferencing the Resource</h2>
 						<p class="issue">TODO: Specify the algorithm for processing the `versionTime` DID parameter.</p>
 					</li></ol>
 				</li>
-				<li>If the <var>input <a>DID URL</a></var> contains the
-				<a href="https://www.w3.org/TR/did-core/#did-parameters">
-				DID parameter</a> <code>service</code> and optionally the <code>relativeRef</code> DID parameter:
-				<pre class="example nohighlight">did:example:1234?service=files&relativeRef=%2Fmyresume%2Fdoc%3Fversion%3Dlatest</pre>
-				<ol class="algorithm">
-					<li>From the <var>resolved <a>DID document</a></var>, select the
-					<a href="#service-endpoint-construction">service endpoint</a> whose <code>id</code>
-					property contains a fragment which matches the value of the <code>service</code> DID parameter of the
-					<var>input <a>DID URL</a></var>. This is called the <var>input <a>service endpoint</a></var>.</li>
-					<li>Execute the <a href="#service-endpoint-construction">Service Endpoint Construction</a> algorithm:
+				<li>If the <var>input <a>DID URL</a></var> contains the <a href="https://www.w3.org/TR/did-core/#did-parameters">
+					DID parameter</a> <code>service</code> and/or the <a href="https://www.w3.org/TR/did-core/#did-parameters">
+					DID parameter</a> <code>serviceType</code>, and optionally the
+					<a data-cite="did-core#did-parameters">DID parameter</a> <code>relativeRef</code>:
+					<pre class="example nohighlight">did:example:1234?service=files&relativeRef=%2Fmyresume%2Fdoc%3Fversion%3Dlatest</pre>
 					<ol class="algorithm">
-						<li>Read the value of the <code>serviceEndpoint</code> property of the
-						<var>input <a>service endpoint</a></var>. This is called the <var>input <a>service endpoint</a> URL</var>.</li>
-						<li>Pass the <var>input <a>DID URL </a></var> and <var>input <a>service endpoint</a> URL</var> to
-						the <a href="#service-endpoint-construction">Service Endpoint Construction</a> algorithm.</li>
-						<li>The result is called the <var>output <a>service endpoint</a> URL</var>.</li>
-					</ol>
+						<li>From the <var>resolved <a>DID document</a></var>, select all
+							<a data-cite="did-core#services">services</a> which fulfill the following conditions:
+							<ul>
+								<p>If the <var>input <a>DID URL</a></var> contains the
+									<a data-cite="did-core#did-parameters">DID parameter</a> <code>service</code>:
+									Select the <a data-cite="did-core#services">service</a> if its <code>id</code>
+									property contains a fragment which matches the value of the <code>service</code> DID parameter.</p>
```

Comment:
Wondering if it might also be worth addressing this bug in this PR - https://github.com/w3c/did-resolution/issues/97

I was working on it, then remembered this PR. So I will at least wait until this is in. But maybe we could just address here?

Reply:
I tried to address this in https://github.com/w3c/did-resolution/pull/125/commits/cda94f2b5c9cb12bba2fb476eaa8fcc487ffc8d0, i.e. the value of the service parameter can be an absolute or relative URL.